GetMalshab/index.ts

```diff
@@ -37,6 +39,10 @@ const getNodesOfMalshabEvents = async (malshabIdentityNumber: String, populteBy?
     return nodesOfMalshabEvents;
 };
 
+const filterMalshabAttachments = (user: User, malshab: Malshab) => {
+    return malshab.attachments?.filter((attachmentKey) => isAttachmentKeyMatchingUserClassification(attachmentKey, user));
+};
```

Comment:
Convert to inline arrow function:

```suggestion
const filterMalshabAttachments = (user: User, malshab: Malshab) => malshab.attachments?.filter((attachmentKey) => isAttachmentKeyMatchingUserClassification(attachmentKey, user));
```

Reply:
no need, doesn't add any readability or performance